source/spirv_target_env.cpp

```diff
@@ -155,31 +197,43 @@ bool spvReadEnvironmentFromText(std::vector<char>& text, spv_target_env* env) {
       // Try to match against the expected version string
       constexpr const char* kVersionPrefix = "; Version: 1.";
       constexpr const auto kPrefixLength = 13;
-      constexpr const auto kMinimumVersionLength = kPrefixLength + 2;
-      if (i + kMinimumVersionLength >= text.size()) return false;
+      // Where do we expect to find the minor version digit.
```

Comment:
Is this a question or a statement?

Reply:
heh. it's a statement

Reply:
Changed the comment to `'minor_digit_pos' is the expected position of the version digit. `